tests/reference/asr-bindc_03-95dbba7.stderr

```diff
@@ -3,3 +3,7 @@ semantic error: Target type specified in c_p_pointer must have deferred shape.
   |
 6 |     A: Pointer[i16[:]] = c_p_pointer(b, i16[n * k], array([k * n]))
   |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
+
+
+Note: Please report unclear or confusing messages as bugs at
+https://github.com/lcompilers/lpython/issues.
```

Comment:
All error reference tests now have this text, it used to be disabled via `--no-error-banner`.

Reply:
Separate issue. Same thing is there in LFortran.